pkg/util/azureclient/mgmt/storage/accounts.go

```diff
@@ -6,14 +6,16 @@ package storage
 import (
 	"context"
 
-	mgmtstorage "github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2019-04-01/storage"
+	mgmtstorage "github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2019-06-01/storage"
 	"github.com/Azure/go-autorest/autorest"
 
 	"github.com/Azure/ARO-RP/pkg/util/azureclient"
 )
 
 // AccountsClient is a minimal interface for azure AccountsClient
 type AccountsClient interface {
+	GetProperties(ctx context.Context, resourceGroupName string, accountName string, expand mgmtstorage.AccountExpand) (result mgmtstorage.Account, err error)
+	Update(ctx context.Context, resourceGroupName string, accountName string, parameters mgmtstorage.AccountUpdateParameters) (result mgmtstorage.Account, err error)
```

Comment:
For anyone wondering why we adding this and not using - they will be used later. See #1657 (`pkg/operator/controllers/storageaccounts/storageaccounts.go`).